runtime/core.go

```diff
@@ -893,7 +893,62 @@ func (op compareOp) slot(t *Type) *binaryOpSlot {
 }
 
 func compareRich(f *Frame, op compareOp, v, w *Object) (*Object, *BaseException) {
-	// TODO: Support __cmp__.
+	r, raised := tryRichCompare(f, op, v, w)
+	if raised != nil {
+		return nil, raised
+	}
+	if r != NotImplemented {
+		return r, nil
+	}
+	return try3wayToRichCompare(f, op, v, w)
+}
+
+// convert3wayToObject converts the integer results from a 3-way
+// comparison to a suitable boolean value for the given rich
+// comparison op.
+func convert3wayToObject(op compareOp, c int) *Object {
+	b := false
+	switch op {
+	case compareOpLT:
+		b = c < 0
+	case compareOpLE:
+		b = c <= 0
+	case compareOpEq:
+		b = c == 0
+	case compareOpNE:
+		b = c != 0
+	case compareOpGE:
+		b = c >= 0
+	case compareOpGT:
+		b = c > 0
+	}
+	if b {
```

Comment:
I always use `make cover` to check for coverage results, but that seems to give inconsistent results.  I will try the command you posted above.  Thanks.